pkg/cmd/attestation/verify/verify.go

```diff
@@ -151,7 +151,6 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 				}
 				config.TrustDomain = td
 				opts.Tenant = tenant
-				opts.OIDCIssuer = fmt.Sprintf(GitHubTenantOIDCIssuer, tenant)
```

Comment:
This detection of the tenancy and defaulting of the tenant-specific issuer string now happens in the `VerifyCertExtensions` function.